README.md

```diff
@@ -1 +1,191 @@
-# LeaderBoard-new
\ No newline at end of file
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
```

Comment:
![image](https://github.com/AnsarIbrahim/LeaderBoard-new/assets/93039359/f058939c-4330-49a9-84e0-ec6015362c83)

- [ ] It would be nice if you fix the links highlighted in the image above too because these are broken 😨 Kindly fix them or you can remove any of them (if these links don't have any section) to make your readme descriptive and professional 😊 👍